src/ibex_bluesky_core/callbacks/__init__.py

```diff
@@ -229,7 +229,7 @@ def start(self, doc: RunStart) -> None:
                     # where a fit result can be returned before
                     # the QtAwareCallback has had a chance to process it.
                     self._subs.append(self._live_fit)
-                self._subs.append(LiveFitPlot(livefit=self._live_fit, ax=ax))
+                self._subs.append(LiveFitPlot(livefit=self._live_fit, ax=ax, num_points=5000))
```

Comment:
this is a lot more than the default of 100. could you add a comment as to why pls?